src/app/static/src/js/components/runWorkflow/runWorkflowProgress.vue

```diff
@@ -81,6 +81,7 @@ interface Data {
     workflowRunStatus: null | WorkflowRunStatus;
     error: string;
     defaultMessage: string;
+    polling: null | number;
```

Comment:
I don't feel strongly about this, but `polling` implies a boolean flag to me. `timer` or similar might be more appropriate.

Reply:
We called it `pollingTimer` in runningReportDetails, would be consistent for these to match